policies/releasestrat.html

```diff
@@ -55,65 +82,92 @@ <h5>
+	  <p>
+	    For any major or minor release, we have defined the following
+	    release criteria:
+	  </p>
+
 	  <ul>
-	    <li>All open github issues/PRs older than 2 weeks at the time of release
-	    to be assessed for relevance to 1.1.1. Any flagged with the 1.1.1
-	    milestone to be closed (see below)</li>
-	    <li>Clean builds in Travis and Appveyor for two days</li>
-	    <li>run-checker.sh to be showing as clean 2 days before release</li>
-	    <li>No open Coverity issues (not flagged as "False Positive" or "Ignore")</li>
-	    <li>TLSv1.3 RFC published (with at least one beta release after the publicaction)</li>
+	    <li>
+	      All open github issues/PRs older than 2 weeks at the time of
+	      release to be assessed for relevance to the version to be
+	      released. Any flagged with the a milestone for the version
+	      to be released must be closed (see below)
+	    </li>
+	    <li>
+	      Clean builds in Travis and Appveyor for two days
+	    </li>
+	    <li>
+	      run-checker.sh to be showing as clean 2 days before release
+	    </li>
+	    <li>
+	      No open Coverity issues (not flagged as "False Positive" or
+	      "Ignore")
+	    </li>
 	  </ul>
 
-	  <p>Valid reasons for closing an issue/PR with a 1.1.1 milestone might be:</p>
+	  <p>
+	    Valid reasons for closing an issue/PR with a milestone for the
+	    version might be:
+	  </p>
+
 	  <ul>
-	    <li>We have just now or sometime in the past fixed the issue</li>
-	    <li>Unable to reproduce (following discussion with original reporter if
-	    possible)</li>
-	    <li>Working as intended</li>
-	    <li>Deliberate decision not to fix until a later release (this wouldn't
-	    actually close the issue/PR but change the milestone instead)</li>
-	    <li>Not enough information and unable to contact reporter</li>
-	    <li>etc</li>
+	    <li>
+	      We have just now or sometime in the past fixed the issue
+	    </li>
+	    <li>
+	      Unable to reproduce (following discussion with original reporter
+	      if possible)
+	    </li>
+	    <li>
+	      Working as intended
+	    </li>
+	    <li>
+	      Deliberate decision not to fix until a later release (this
+	      wouldn't actually close the issue/PR but change the milestone
+	      instead)
+	    </li>
+	    <li>
+	      Not enough information and unable to contact reporter
+	    </li>
 	  </ul>
+
+	  <hr />
+
+	  <p>No API or ABI breaking changes are allowed in a minor or patch release.
+	  The following stability rules apply to all changes made to code
+	  targeted for a major release from version 3.0.0 or later:</p>
+	  <ul>
+	    <li>No existing public interface can be modified except where changes
+	    are unlikely to break source compatibility.
+	    <li>No existing public interface can be removed until its replacement
+	    has been in place in an LTS stable release and the original function has
+	    been documented as deprecated for at least 5 years. A public interface
```

Comment:
The mixture of "public interface" and "original function" disturbs me, and "original function" feels linguistically clumsy.  May I suggest the following reprase?

``` html
	    <li>No existing public interface can be removed until it has been
            documented as deprecated for at least 5 years and its replacement
            has been in place in an LTS stable release. A public interface
```

Reply:
I'm not super-picky about which wording, but I wonder if we could get away with 3 or even 2 years.

Reply:
2 or 3 years sounds quite short to me.

Reply:
Regardless of length of time (I'm fine with 5 years), I'll still pick at the phrasing :wink:

Reply:
I don't mind shortening the time...  but oh, this comes back to phrasing again!  I assumed that the 5 years only referred to deprecation, is that correct?  With the current phrasing, "at least 5 years" could be understood to refer both to the deprecation and the presence in an LTS stable release.

Regardless of phrasing, I'd apply much more importance in the replacement's presence in a LTS release than the length of deprecation.

Reply:
I updated the phrasing to try and make this clearer.